sklearn/metrics/classification.py

```diff
@@ -2007,3 +2007,148 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
     y_true = np.array(y_true == pos_label, int)
     y_true = _check_binary_probabilistic_predictions(y_true, y_prob)
     return np.average((y_true - y_prob) ** 2, weights=sample_weight)
+
+
+def calibration_loss(y_true, y_prob, sample_weight=None, reducer="sum",
+                     n_bins=10, sliding_window=False, normalize=True,
+                     pos_label=None):
+    """Compute  calibration loss.
```

Comment:
2 spaces before calibration

Reply:
reading the suggested papers is there a recommended default for the parameters?
for example I would expect the sliding_window=True option to be recommended
by default.